commit 13a6a27eecdd1fb527b9151309366970b182a58d
Author: Tomas Bzatek <tbzatek@redhat.com>
Date:   Thu Oct 20 17:17:10 2022 +0200

    tests: Fix LIO target config auth
    
    Linux kernel 6.0 brought number of the LIO target changes related to authentication
    that made our tests fail. Turned out our target config was incorrect, e.g.
    not requiring auth for CHAP tests, etc. The kernel 6.0 looks to be more strict
    in this regard.

diff --git a/src/tests/dbus-tests/targetcli_config.json b/src/tests/dbus-tests/targetcli_config.json
index 25d506b6..3be9eac2 100644
--- a/src/tests/dbus-tests/targetcli_config.json
+++ b/src/tests/dbus-tests/targetcli_config.json
@@ -385,7 +385,7 @@
       "tpgs": [
         {
           "attributes": {
-            "authentication": 0,
+            "authentication": 1,
             "cache_dynamic_acls": 0,
             "default_cmdsn_depth": 64,
             "default_erl": 0,
@@ -432,7 +432,7 @@
             }
           ],
           "parameters": {
-            "AuthMethod": "CHAP,None",
+            "AuthMethod": "CHAP",
             "DataDigest": "CRC32C,None",
             "DataPDUInOrder": "Yes",
             "DataSequenceInOrder": "Yes",
@@ -471,7 +471,7 @@
       "tpgs": [
         {
           "attributes": {
-            "authentication": 0,
+            "authentication": 1,
             "cache_dynamic_acls": 0,
             "default_cmdsn_depth": 64,
             "default_erl": 0,
@@ -520,7 +520,7 @@
             }
           ],
           "parameters": {
-            "AuthMethod": "CHAP,None",
+            "AuthMethod": "CHAP",
             "DataDigest": "CRC32C,None",
             "DataPDUInOrder": "Yes",
             "DataSequenceInOrder": "Yes",
